code/game/machinery/computer/camera_advanced.dm

```diff
@@ -65,6 +63,14 @@
 		give_eye_control(L)
 		eyeobj.setLoc(eyeobj.loc)
 
+/obj/machinery/computer/camera_advanced/attack_robot(mob/user)
+	if(!Adjacent(user)) //Borgs can use the console without issue so long as they remain next to it.
```

Comment:
It is not a powergaming thing. The console boots anyone not adjacent to the console in `check_eye()`, and I did not want to add another check to that line.